bin/migrate-oats-data/applications/submissions/sql/parcels/primary_contact/application_submission_primary_contact_count.sql

```diff
@@ -0,0 +1,4 @@
+SELECT count(*)
+FROM oats.oats_alr_application_parties oaap
+    JOIN alcs.application_submission apps ON apps.file_number::bigint = oaap.alr_application_id
+    JOIN alcs.application_owner appo ON appo.oats_application_party_id = oaap.alr_application_party_id;
```

Comment:
Nitpick: I'm getting a nondescript no entry sign on this file. I think it may be due to a missing semicolon.

Reply:
Semicolon exists on line 4, I think we can safely ignore the newline